### ISSUE TITLE
Question: Does IPBan work with MultiOTP?

### ISSUE THREAD

**Issue by levihaynes** [DigitalRuby/IPBan#53]
Since I've installed MultiOTP on a Windows 10 PC (1809) I see lots of RDP failed connections but they don't appear to be getting blocked, or at least not in the log file.

Do you know if the two are incompatible?

https://github.com/multiOTP/multiotp/wiki

**Comment by jjxtra** [comment]
Can you post the raw xml from the event viewer?

**Comment by levihaynes** [comment]
After doing some research I believe the problem is related to the event log not saving the client ip address for known usernames. I refer to this article http://purerds.org/remote-desktop-security/auditing-remote-desktop-services-logon-failures-1/

The problem with MultiOTP is that it allows the hacker to know what the username is which means IPBan doesn't receive the ip address. There might be a work-a-round but I'll leave you to decide if its worth investing your time over.

**Comment by jjxtra** [comment]
I just did a test on my server 2012 box with my user name and failed password and it showed up in both the standard security log with user name, along with the 2012+ only rdpcorets log. I have ntlm login enabled which I know is not secure but I like gathering hacker ip addresses and blocking them :) NLA is turned off. In fact if I turn NLA on, I am unable to remote in to the box at all with Windows 10. I learned that the hard way and had to use a backup ssh connection which luckily I had openssh installed.
